llvm/include/llvm/TargetParser/Triple.h

```diff
@@ -1119,7 +1119,9 @@ class Triple {
 
   /// True if the target supports both general-dynamic and TLSDESC, and TLSDESC
```

Comment:
This comment is not precisely true of all `isOSFuchsia()` cases. That is, it says this is true only if the target supports non-TLSDESC as a non-default option.  That's not true for `aarch64-*`, so it's not true for `*-fuchsia`.
If that's actually fine for how this is used, then the comment here should be changed.

Reply:
Yeah, the comment here is a bit inaccurate. I've updated it to reflect how it's used in practice. Our Aarch64 implementation does not account for non-TLSDESC, and thus does not check if TLSDESC is enabled, like we do for other backends.